features/FEATURE_BLE/ble/generic/GenericSecurityManager.h

```diff
@@ -26,7 +26,7 @@
 #include "ble/pal/SigningEventMonitor.h"
 #include "ble/generic/GenericGap.h"
 #include "ble/pal/PalSecurityManager.h"
-#include "ble/ArrayView.h"
+#include "platform/Span.h"
```

Comment:
Wait, since we imported Span to ble namespace, maybe we don't need/shouldn't use platform/Span.h?

Reply:
cleaning up includes can be part of another task I believe.

Reply:
This include (platform/Span.h) is introduced by this PR, so I think it's better to make sure things we do here are correct?

Reply:
True. We can make sure it is included through `BLEType.h`

Reply:
It should look better now 😛